src/action_builder.js

```diff
@@ -587,7 +587,12 @@ module.exports = class ActionBuilder {
     this._modules.forEach((mod) => {
       archive.directory(path.resolve(this._cwd, `node_modules/${mod}`), `node_modules/${mod}`);
     });
+
+    archive.directory(path.resolve(__dirname, '../node_modules/node-fetch'), 'node_modules/node-fetch');
```

Comment:
hmm...this should be included automatically if imported somewhere in the code....

Reply:
The `index.js` wrapper is never processed by webpack and even if the action imports `node-fetch`, it won't get picked up, hence the brute-force approach.

Reply:
> even if the action imports `node-fetch`, it won't get picked up, hence the brute-force approach.

but this would be wrong then.....